### PR TITLE
Adding TCP tests for Cloud SQL samples

### PR DESCRIPTION
**Pull request by craiglabenz**
Added TCP connectivity tests for Cloud SQL

## Description

Fixes #<ISSUE-NUMBER>

Note: It's a good idea to open an issue first for discussion.

## Checklist
- [ ] **Tests** pass
- [ ] **Lint** pass: `bundle exec rubocop`
- [ ] Please **merge** this PR for me once it is approved.

### TODOs before tests can pass:
- [ ] Populate 3 database names in secrets file
- [ ] Assure SQLServer user's password is correct (may require creating instance)

### CODE REVIEW

**Comment by craiglabenz** [comment]
Are either @shubha-rajan, @dazuma, or @TheRoyalTnetennba aware of what might be causing builds against this PR to fail? I'm sure it's my edits to  `spec/kokoro-run-all.sh`, but I have so far been unsuccessful isolating a specific error string in the logs.

**Comment by shubha-rajan** [comment]
> Are either @shubha-rajan, @dazuma, or @TheRoyalTnetennba aware of what might be causing builds against this PR to fail? I'm sure it's my edits to `spec/kokoro-run-all.sh`, but I have so far been unsuccessful isolating a specific error string in the logs.

Looking at the logs, it doesn't seem like the Cloud SQL tests are failing. The only failures I see (in both Ruby 2.4 and Ruby 2.5) are for servicedirectory.

**Comment by shubha-rajan** [comment]
Errors I'm seeing in the logs: 
```
An error occurred while loading ./spec/controllers/votes_spec.rb.
Failure/Error: ActiveRecord::Migration.maintain_test_schema!

PG::ConnectionBad:
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?
# /usr/local/bundle/gems/pg-0.21.0/lib/pg.rb:56:in `initialize'
# /usr/local/bundle/gems/pg-0.21.0/lib/pg.rb:56:in `new'
# /usr/local/bundle/gems/pg-0.21.0/lib/pg.rb:56:in `connect'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_adapters/postgresql_adapter.rb:692:in `connect'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_adapters/postgresql_adapter.rb:223:in `initialize'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_adapters/postgresql_adapter.rb:48:in `new'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_adapters/postgresql_adapter.rb:48:in `postgresql_connection'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:830:in `new_connection'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:874:in `checkout_new_connection'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:853:in `try_to_checkout_new_connection'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:814:in `acquire_connection'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:538:in `checkout'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:382:in `connection'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:1033:in `retrieve_connection'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_handling.rb:118:in `retrieve_connection'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_handling.rb:90:in `connection'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/migration.rb:583:in `load_schema_if_pending!'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/migration.rb:599:in `block in maintain_test_schema!'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/migration.rb:848:in `suppress_messages'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/migration.rb:604:in `method_missing'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/migration.rb:599:in `maintain_test_schema!'
# ./spec/rails_helper.rb:28:in `<top (required)>'
# ./spec/controllers/votes_spec.rb:15:in `require'
# ./spec/controllers/votes_spec.rb:15:in `<top (required)>'
No examples found.

Finished in 0.00006 seconds (files took 0.81945 seconds to load)
0 examples, 0 failures, 1 error occurred outside of examples
```

```
Failure/Error: ActiveRecord::Migration.maintain_test_schema!

Mysql2::Error::ConnectionError:
  Can't connect to local MySQL server through socket '/cloudsql/' (2)
# /usr/local/bundle/gems/mysql2-0.5.3/lib/mysql2/client.rb:90:in `connect'
# /usr/local/bundle/gems/mysql2-0.5.3/lib/mysql2/client.rb:90:in `initialize'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_adapters/mysql2_adapter.rb:22:in `new'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_adapters/mysql2_adapter.rb:22:in `mysql2_connection'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:830:in `new_connection'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:874:in `checkout_new_connection'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:853:in `try_to_checkout_new_connection'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:814:in `acquire_connection'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:538:in `checkout'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:382:in `connection'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:1033:in `retrieve_connection'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_handling.rb:118:in `retrieve_connection'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/connection_handling.rb:90:in `connection'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/migration.rb:583:in `load_schema_if_pending!'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/migration.rb:599:in `block in maintain_test_schema!'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/migration.rb:848:in `suppress_messages'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/migration.rb:604:in `method_missing'
# /usr/local/bundle/gems/activerecord-5.2.4.4/lib/active_record/migration.rb:599:in `maintain_test_schema!'
# ./spec/rails_helper.rb:28:in `<top (required)>'
# ./spec/controllers/votes_spec.rb:15:in `require'
# ./spec/controllers/votes_spec.rb:15:in `<top (required)>'
No examples found.

Finished in 0.00006 seconds (files took 1.86 seconds to load)
0 examples, 0 failures, 1 error occurred outside of examples
```

File: cloud-sql/mysql/activerecord/bin/run_tests [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#! /bin/sh
+
+# Run the tests with default values for all
+# environment variable
```
Comment (shubha-rajan) [inline cloud-sql/mysql/activerecord/bin/run_tests:4]: Kokoro still thinks the tests are passing, even though there are connection errors in the logs. Maybe try adding `set -e` to these scripts to fix it?
image

**Comment by shubha-rajan** [comment]
@TheRoyalTnetennba @dazuma Can we get a review for this?

**Review by TheRoyalTnetennba — APPROVED** [review]

**Pull request merged**